melodic/distribution.yaml

```diff
@@ -4687,6 +4687,26 @@ repositories:
       type: git
       url: https://github.com/inertialsense/inertial_sense_ros.git
       version: master
+  inno_sim_interface:
+    doc:
+      type: git
+      url: https://github.com/InnopolisAero/inno_sim_interface.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/InnopolisAero/inno_sim_interface.git
+      version: master
+    status: maintained
+  innopolis_vtol_dynamics:
+    doc:
+      type: git
+      url: https://github.com/InnopolisAero/innopolis_vtol_dynamics.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/InnopolisAero/innopolis_vtol_dynamics.git
```

Comment:
@frontw [`innopolis_vtol_dynamics`](https://github.com/InnopolisAero/innopolis_vtol_dynamics/tree/master/uav_dynamics/inno_vtol_dynamics)'s package.xml also needs an update: it declares an MIT license on a GPLv3 licensed repository, it does not reflect all dependencies actually used.

Reply:
There's also something to be said about a repository this full of submodules, with ROS packages themselves, some from third-party sources. IIRC those will not be fetched during package scrapping, but I'm not so sure about the licensing. CC @clalancette @tfoote for policy in these cases.

Reply:
Yes, please submit the actual upstream package individually instead of a meta repository with a lot of submodules so that everyone can be aware of where sources are coming from upon inspection w/o having to resolve and traverse the submodules. (There are also tools in our toolchain which will fail on submodules too so that's better too.)